include/vcpkg/base/messages.h

```diff
@@ -1081,6 +1113,10 @@ namespace vcpkg
                     "unknown binary provider type: valid providers are 'clear', 'default', 'nuget', "
                     "'nugetconfig','nugettimeout', 'interactive', 'x-azblob', 'x-gcs', 'x-aws', "
                     "'x-aws-config', 'http', and 'files'");
+    DECLARE_MESSAGE(UnknownParameterForIntegrate,
+                    (msg::value),
+                    "'{value}' is the parameter",
```

Comment:
```suggestion
                    "'{value}' is a user-supplied command line option. For example, given vcpkg integrate frobinate, {value} would be frobinate.",
```

Feel free to use another nonsense word than frobinate :)